README.md

```diff
@@ -7,79 +9,41 @@ One Paragraph of project description goes here
 
-Explain the desired workflow. Workflow may deviate from the instructions below depending on the projects needs.
-
-**Instructions below are for developers actively working on the repo.**
-
-An issue will be assigned to you via Github. Your workflow begins after assignment:
-
-1. Create a branch with the issue number as the branch name (e.g. `134`)
-2. Move issue into 'In Progress' column of project board
-3. Do the work
-    - While you work, you may wish to have the app running with live reload. Run `ng serve` to do so.
-4. Write tests for the work
-    - This is a critical step. STNWeb2 uses the [Jasmine library](https://jasmine.github.io/) for unit testing. Please refer there for docs and examples. Also refer to the [Angular testing guide](https://angular.io/guide/testing).
-5. Run the tests to be sure they all pass and that the overall thresholds are met: `ng test`
-6. Be sure the app runs in the browser without errors: `ng serve`
-7. Ensure the app builds without error: `ng build`
-    - If any of the checks above fail, please fix the issue. Ask for help if needed.
-8. Once all checks have passed and you are ready to submit a Pull Request, update the changelog with a brief description of what your work added, changed, or fixed. There is an Unreleased section at top with subheadings for each category. Edit the file CHANGELOG.md found at project root.
-9. Add the changed files `git add .` and commit `git commit -m '[your commit message here]'` you commit message should reference the issue number and include a very brief description of the work.
-10. **Pull from `dev`**
-    - Run `git pull origin dev`. This is a critical step. It ensures your Pull Request is synced with the latest work in the main `dev` branch. If you are lucky, it will auto-merge. Otherwise, you may have to resolve conflicts between your commit and what currently exists in dev. Please be careful with this step so no code is lost - ask for help if you are unsure what to do.
-    - If manually merging, you will have changed files so you will need to add and commit once more (see step 9).
-11. Push your committed and synced branch to the remote repo (Github): `git push origin [your branch name]`
-12. Submit Pull Request (PR) to merge your issue branch into `dev`
-    - Automated checks will be run against your PR. Failure of any of these will mean you need to re-commit to your branch to update your PR with the fix. A colleague will review your work and either approve it or request changes. Upon approval of the PR your issue will be automatically moved to the 'In Dev Branch' column of the project board.
-
-Move onto next assigned issue and start back at step 1.
+Explain the desired workflow for development. This should be outlined by the lead developer of the repository, based on the guidelines and requirements laid out in the [WIM Collaborative Development Constitution](https://github.com/USGS-WiM/wimfo/tree/dev/Development/Development_at_WIM/Application_Standards).
```

Comment:
```suggestion
Explain the desired workflow for development. This should be outlined by the lead developer of the repository, based on the guidelines and requirements detailed in the [WIM Collaborative Development Constitution](https://github.com/USGS-WiM/wimfo/tree/dev/Development/Development_at_WIM/Application_Standards).
```

'Laid out' feels weird for some reason, feel free to ignore if you think it's better that 'detailed' or 'outlined' or something